app/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en" ng-app="webshopModule">
+<html lang="en" ng-app="webshopModule" ng-controller="ProductListCtrl">
```

Comment:
This logic is specific to the product listing. Please specify it in the routing configuration.